Sources/PayPalUI/PaymentButtonColor.swift

```diff
@@ -1,14 +1,107 @@
 import UIKit
 
-enum PaymentButtonColor {
-    case gold, darkBlue
+public enum PaymentButtonColor: String {
 
-    var rawValue: UIColor? {
+    /// The gold background and blue wordmark, monogram, and black text.
+    case gold
+
+    /// The white background and blue wordmark, monogram, and black text.
+    case white
+
+    /// The black background and monochrome wordmark, monogram, and white text.
+    case black
+
+    /// The silver background and blue wordmark, monogram, and black text.
+    case silver
+
+    /// The blue background and white wordmark, blue monogram, and white text.
+    case blue
+
+    /// The dark blue background with PayPal Credit wordmark and monogram.
+    case darkBlue
+
+    var color: UIColor {
+        switch self {
+        case .gold:
+            return UIColor(hexString: "#FFC439")
+
+        case .white:
+            return UIColor(hexString: "#FFFFFF")
+
+        case .black:
+            return UIColor(hexString: "#000000")
+
+        case .silver:
+            return UIColor(hexString: "#EEEEEE")
+
+        case .blue:
+            return UIColor(hexString: "#0070BA")
+
+        case .darkBlue:
+            return UIColor(hexString: "#073990")
+        }
+    }
+
+    var fontColor: UIColor {
+        switch self {
+        case .gold, .white, .silver:
+            return .black
+
+        case .blue, .black, .darkBlue:
+            return .white
+        }
+    }
+
+    public var description: String {
         switch self {
         case .gold:
-            return UIColor(named: "PayPalGold", in: PaymentButton.bundle, compatibleWith: nil)
```

Comment:
I think we can remove these assets as well now that we are using the hex colors directly?